### DISCUSSION TITLE
Trying to duplicate a transaction made with GUI using go web3

### DISCUSSION THREAD

**Discussion by 621625** [gochain/web3#229] [Q&A]
Hello,

I'm BM. A very nice and polite guy.

I've been trying to duplicate a transaction that I made with a GUI in my Terminal for a while without success.

This is the transaction I'm trying to duplicate:

https://polygonscan.com/tx/0xcaf7358788aed4fd73b4fc26717e44a54750811fc1c62e6997db9bb0ae9f7585

Could anyone please show me a working example on how to replicate this transaction using go web3?

**Comment by treeder** [comment]
Snag the input data from the explorer:

![image](https://user-images.githubusercontent.com/75826/136435666-db7f4863-9d05-473a-99ed-b7e6f8bf1661.png)

Then use `web3 contract call` with `--data COPIED_DATA`.
